src/components/Table.tsx

```diff
@@ -176,7 +176,7 @@ class TableRow<T> extends React.Component<TableRowProps<T>, {}> {
 							data-key={keyAttribute}
 							onClick={this.props.onRowClick}
 							{...cellAttributes}
-							key={column.field}
+							key={`${column.field}-${idx}`}
```

Comment:
We shouldn't really be using the index of the column in the key. This is a little better than using the index alone in the key (i.e. `key={idx}`), but still has the same problems for the cases where you do have two columns on the same field. 

https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318 is good background on how this can go wrong and how to fix it in the more general case. With this, if you do somehow swap the position of two columns with the same field, then React won't notice and you might get the wrong state, and when you change the index of a column in general you'll have to rebuild the whole component, even if that's unnecessary. It's not a huge problem, but it is the kind of thing that could catch us out painfully later on.

What is this case where we have two columns for the same data like this? Is there some other property or difference between them we could use?

Reply:
@pimterry  This is the case that I'm working on. Both these columns have to access the public_key in order to display their content.

```js
{
	label: 'Fingerprint',
	field: 'public_key',
	sortable: false,
	render: (publicKey: ResinSdk.SSHKey['public_key']) => (
		<Txt>{getSSHKeyFingerPrint(publicKey)}</Txt>
	),
},
{
	label: 'Key',
	field: 'public_key',
	sortable: false,
	render: (publicKey: ResinSdk.SSHKey['public_key'], key) => (
		<Flex>
			<PublicKeyField onClick={() => this.handleKeyDisplay(key)}>
				{publicKey}
			</PublicKeyField>
			<TextWithCopy copy={publicKey} showCopyButton="always" />
		</Flex>
	),
},
```

Reply:
Another approach is to expose a `key` property in order to hardcode another value